amodevices/hvs_controller/hvs_controller.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+"""
+@author: Jack Mango/Berkeley
+
+Device driver for Matsusada J4-5P and KA-10P high voltage supplies.
+J4-5P and KA-10P high voltage supply output voltages can be set using an analog
+control voltage and measured using an analog monitor voltage. Additionally,
+KA-10P high voltage supply current can be measured using a monitor voltage. These
+analog voltages are using a National Instruments 9205 ADC and are set using a 
+National Instruments 9264 DAC.
+
+The controller implements a caching scheme for current and voltage measurement. If
+it's been over a cache_interval since the last measurement the controller will perform
+another one for all channels.
+
+Measurement of monitor signals and setting of control signals is implemented using two
+separate PyDAQmx Tasks. There is an assumed ordering to the high voltage supplies set 
+in the config file: The first four entries in the 'Monitors' dictionary should be for
+the voltage monitors, channels one through four. The last two should be the current
+monitors for the first, followed by the fourth high voltage supply. The 'Controls'
+should also be ordered the same as the first four monitors, in increasing order.
+"""
```

Comment:
You should include an example of the config file in the repo if it is required for the code to run.

The scheme of handling current-reading ability is a bit too specific and does not generalize well - it's to be able to define whether current can be read out for each channel instead of fixing it for channel 1 and 4. We want to be able to use this code for different combinations of different models of HV supplies.

I suggest that in the config, you define a structure handling the channels with each channel a different entry. This could be dict-like JSON files or, probably better, a TOML file (use `tomli` library), which also translates into a dict. Then, for each channel, there is an entry defining the type of HV supply - for now this would only be Matsusada J4-5P and KA-10P. Then, there are other entries for that channel defining which ADC/DAC channels to use.